test/e2e/cli_test.go

```diff
@@ -281,6 +281,31 @@ func (s *CLISuite) TestTemplate() {
 			}
 		})
 	})
+	s.Run("Template Submittable", func() {
```

Comment:
```suggestion
	s.Run("Submitable Template", func() {
```

Reply:
test names don't typically have spaces in them either